ci/build.ps1

```diff
@@ -13,17 +13,19 @@ $nvimCmakeVars = @{
   BUSTED_OUTPUT_TYPE = 'nvim';
 }
 
+# For pull requests, skip some build configurations to save time.
+if ($env:APPVEYOR_PULL_REQUEST_HEAD_COMMIT
+    -and $env:CONFIGURATION -match '^MSVC_64|MINGW_32$') {
```

Comment:
Keeping in mind I know nothing about Powershell scripting, this regex looks odd.  I would expect something like `^(MSVC_64|MINGW_32)$`, otherwise the `^` anchor only applies to `MSVC_64` and the `$` anchor only applies to `MINGW_32`.